src/main/java/com/wire/bots/roman/MessageHandler.java

```diff
@@ -54,6 +54,7 @@ public boolean onNewBot(NewBot newBot, String auth) {
         message.userId = newBot.origin.id;
         message.handle = newBot.origin.handle;
         message.locale = newBot.locale;
+        message.token = newBot.token;
```

Comment:
Oh.. this `token`?! Well, this is not roman's token its Wire's bot token. I think this token should never be exposed to end user. Instead it would rather expose additional roman endpoint that accepts roman token, resolves it to Wire token and just forwards this req to Wire BE with Wire token.